drivebrain_core_impl/drivebrain_estimation/include/StateEstimator.hpp

```diff
@@ -79,6 +78,18 @@ namespace core
         void set_previous_control_output(SpeedControlOut prev_control_output);
 
     private:
+        void _recv_low_level_state(std::shared_ptr<google::protobuf::Message> message);
+        void _recv_inverter_states(std::shared_ptr<google::protobuf::Message> msg);
+
+        template <size_t ind, typename inverter_dynamics_msg>
+        void _handle_set_inverter_dynamics(std::shared_ptr<google::protobuf::Message> msg);
+
+        Tire_Model_Codegen::ExtY_Tire_Model_Codegen_T _eval_estimator(core::VehicleState vehicle_state, core::RawInputData raw_input_data);
+
+        std::shared_ptr<hytech_msgs::VehicleData> _set_tire_dynamics(std::shared_ptr<hytech_msgs::VehicleData> msg_out, Tire_Model_Codegen::ExtY_Tire_Model_Codegen_T res);
```

Comment:
After the new simulink_automation pr gets pulled in these shouldn't be necessary?

Reply:
("these" refers to the methods packing protos)